apps/language_server/lib/language_server/server.ex

```diff
@@ -325,8 +329,11 @@ defmodule ElixirLS.LanguageServer.Server do
     {:ok, %{"capabilities" => server_capabilities()}, state}
   end
 
+  # NOTE: This doesn't seem to match spec
+  # https://github.com/Microsoft/language-server-protocol/blob/master/versions/protocol-2-x.md#shutdown-request
```

Comment:
Can you explain how the implementation doesn't match the spec? To me it looks like it does match.

Reply:
@bfolkens you are looking at an outdated spec, see https://github.com/elixir-lsp/elixir-ls/pull/130#issuecomment-589934892